lara-typescript/src/interactive-api-client/types.ts

```diff
@@ -359,6 +359,7 @@ export interface IAuthInfo {
 export interface IAttachmentUrlRequest extends IBaseRequestResponse {
   name: string;
   operation: "read" | "write";
+  type?: string;    // defaults to application/json
```

Comment:
I think this should be called `contentType` just to help devs know what this better from the name.